examples/tasks/df-influxdb.json

```diff
@@ -0,0 +1,28 @@
+{
+    "version": 1,
+    "schedule": {
+        "type": "simple",
+        "interval": "1s"
+    },
+    "workflow": {
+        "collect": {
+            "metrics": {
+                "/intel/procfs/*": {}
+            },
+            "config": {},
+            "process": null,
+            "publish": [
+                {
+                    "plugin_name": "influxdb",
+                    "config": {
+                        "host": "172.17.0.20",
```

Comment:
I prefer as a `host` either `localhost` or `127.0.0.1` or enviromental variable instead of specific IP address.